lnrpc/routerrpc/router_backend_test.go

```diff
@@ -475,3 +475,80 @@ func testUnmarshalAMP(t *testing.T, test unmarshalAMPTest) {
 		t.Fatalf("test case has non-standard outcome")
 	}
 }
+
+type ExtractIntentTestCase struct {
+	name             string
+	maxPartsInput    uint32
+	shardAmountMsat  uint64
```

Comment:
Nit: Why not keep the naming here from the above code ?

`maxShardAmt, maxParts` ?